bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -105,10 +98,17 @@ <h1>{{ self.page_title() }}</h1>
         include_highres_image=True
       )}}
 
-      <div class="mzp-l-card-half">
-        {{ content_card('card_10') }}
-        {{ content_card('card_11') }}
-      </div>
+      {% call hero(
```

Comment:
The hero is a little narrow because it is being nested inside a `mzp-l-content` div. (this is more obvious on the text side than the image side). Let me know if you want to chat about a fix but I don't think there's a "right" way to fix it so I'm happy to leave the how up to you.

Reply:
I pulled it out of the `.mzp-l-content` wrapper div and put it in its own. I also modified the image to have a high-res variation, match the 3:2 ratio. This overall looks much better: 

 Before: 
<img width="1536" alt="image" src="https://user-images.githubusercontent.com/2692333/96179978-5e7ea080-0ef7-11eb-9bd4-6f3feca85d18.png">

After: 
<img width="1648" alt="image" src="https://user-images.githubusercontent.com/2692333/96179986-60e0fa80-0ef7-11eb-9476-ea60c415b781.png">